services/generic/battery_charge_control.md

```diff
@@ -1,23 +1,27 @@
-### Battery charge controller service
+# Battery Charge Control Service
 
-#### Service name
+> **Important!** Battery charge control service is experimental and is subject to be removed or modified in a backwards in-compatible way.
 
-`battery_charge_ctrl` - represents battery charge controller.
+Battery charging control service is responsible for controlling charging of a household battery / energy storage. See [inverter services](/services/generic/inverter.md) for more 
+information.
+
+## Service name
 
-#### Interfaces
+`battery_charge_ctrl` - represents battery charge controller.
 
-Type | Interface                | Value type | Properties | Description
------|--------------------------|------------|------------|--------------
-out  | evt.meter_ext.report     | float_map  |            | [Extended meter report](#extended-report-object) with up to 17 data points.
-in   | cmd.meter_ext.get_report | null       |            | Request extended report.
-in   | cmd.mode.get_report      | null       |            |
--|||
-in   | cmd.mode.set             | string     |            | Set charge mode.
-out  | evt.mode.report          | string     |            | Charge mode report.
+## Interfaces
 
+| Type | Interface                | Value type | Description                                                              |
+|------|--------------------------|------------|--------------------------------------------------------------------------|
+| out  | evt.meter_ext.report     | float_map  | Returns [extended meter report](/services/generic/meter.md#definitions). |
+| in   | cmd.meter_ext.get_report | null       | Request the extended report.                                             |
+| -    |                          |            |                                                                          | 
+| in   | cmd.mode.get_report      | null       | Requests the charging mode report.                                       |
+| in   | cmd.mode.set             | string     | Sets the charge mode.                                                    |
+| out  | evt.mode.report          | string     | Reports the charging mode.                                               |
 
-#### Service props
+## Service properties
 
-Name            | Value example                                                                  | Description
-----------------|--------------------------------------------------------------------------------|-------------
-`sup_modes`     | idle, charging, discharging                                                    | Supported modes.
+| Name        | Type      | Example                               | Description               |
```

Comment:
Maybe Example column should be Supported values ? As usually it is so.. ? Examples are below the table.

Reply:
Negative. I worked this through. It is always possible to show an example value, but it is not always possible to easily explain a complete range of possible values. We have either description column to touch this subject or definition section to provide in-depth description. While it may sound easy to define numeric range 1-255 or list enum here, it's impossible to define object schema here without linking outside.